__test__/index.test.ts

```diff
@@ -0,0 +1,85 @@
+import { describe, expect, test } from '@jest/globals'
+import app from '../src/app'
+import supertest from 'supertest'
+import build from '../src/db/build'
+import sequelize from '../src/db/connection'
+
+beforeEach(() => build())
+
+describe('all post ', () => {
```

Comment:
no need to rewrite describe for each test, write a single describe for the router and give each test a descriptive name